src/deprecated.jl

```diff
@@ -0,0 +1,234 @@
+# Introduced in 0.6
+# NOTE: Deprecated Imputor docstrings use julia-repl rather than jldoctest since depwarn
+# output isn't consistent across installs.
+Base.@deprecate_binding(
+    AbstractContext,
+    Assertion,
+    false,
+    "Imputation contexts have been replaced with independent Impute.Assertion types.",
+)
+Base.@deprecate Context(; limit=1.0, kwargs...) Threshold(limit) false
+Base.@deprecate WeightedContext(wv; limit=1.0, kwargs...) Threshold(limit; weights=wv) false
+
+"""
+    Fill(; value=mean)
+
+Fills in the missing data with a specific value.
+The current implementation is univariate, so each variable in a table or matrix will
+be handled independently.
+
+[Deprecated] Use Impute.Replace for constants or Impute.Substitue for median/mode substitution.
```

Comment:
We shgould use warning Admonitionsfor this
https://docs.julialang.org/en/v1/stdlib/Markdown/#Admonitions

Also its missing code quotes